hbase_storage/hbase1.1.1/src/main/java/com/splicemachine/storage/RangedClientPartition.java

```diff
@@ -33,6 +33,7 @@
 public class RangedClientPartition extends ClientPartition implements Comparable<RangedClientPartition>{
     private final HRegionInfo regionInfo;
     private final PartitionServer owningServer;
+    private String regionName;
```

Comment:
It might be better to make this field volatile, since several threads could use the same cached partition without synchronization, but it might not be a big deal.